plugin/parser/differ/pg/differ.go

```diff
@@ -163,10 +163,39 @@ func (*diffNode) modifyColumn(alterTableStmt *ast.AlterTableStmt, oldColumn *ast
 			ColumnName: columnName,
 		})
 	}
+	// compare the DEFAULT
+	oldDefault, oldHasDefault := getDefault(oldColumn)
+	newDefault, newHasDefault := getDefault(newColumn)
+	needSetDefault := (!oldHasDefault && newHasDefault) || (oldHasDefault && newHasDefault && oldDefault != newDefault)
```

Comment:
needSetDefault := (!oldHasDefault && newHasDefault) || (oldDefault != newDefault)

Reply:
Consider the cases:
```
oldDefault = "1+2+3", oldHasDefault = true
newDefault = "", newDefault = false
```
then `(!oldHasDefault && newHasDefault) || (oldDefault != newDefault)` is true, but we need to drop default